src/reanimated2/NativeReanimated/NativeReanimated.ts

```diff
@@ -110,9 +110,15 @@ export class NativeReanimated {
   configureLayoutAnimation(
     viewTag: number,
     type: string,
+    sharedTransitionTag = '',
```

Comment:
seems wrong to have a default value for a second to last parameter of a function but not for the last one. Also empty string does not seem like a great default value. It looks like we don't really need this default value at all?